src/apps/api/serializers/queues.py

```diff
@@ -0,0 +1,75 @@
+from rest_framework import serializers
+from queues.models import Queue
+
+from profiles.models import User
+
+
+class OrganizerSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = [
+            'username',
+            'email',
+            'id'
+        ]
+
+
+class QueueSerializer(serializers.ModelSerializer):
+    is_owner = serializers.SerializerMethodField()
+    organizers = serializers.PrimaryKeyRelatedField(queryset=User.objects.all(), many=True, required=False)
+
+    class Meta:
+        model = Queue
+        fields = [
```

Comment:
DRF themselves actually switched to lists on their repo documentation this year, but haven't updated their other docs. This issue and the related PR discussions seem to indicate the reason for this was being consistent with Django, but Django themselves updated to using lists instead of tuples. If you think these should be tuples I'll change them.

https://github.com/encode/django-rest-framework/issues/6604

Reply:
i think tuples make more sense because it should be immutable?

Reply:
That seems to be the heart of the argument in Tyler's referenced DRF issue, they aren't necessarily immutable.

Reply:
That said, we pretty much use tuples everywhere else in the code, and we aren't doing any mutations, sooo? for consistency, tuples, but for what is being argued as "best practice" list... kinda a coin toss?

Reply:
I'll just change to tuples